build.sbt

```diff
@@ -81,6 +81,8 @@ val scalacheckDep = withoutScalaLang("org.scalacheck" %% "scalacheck" % versionN
   * real publishing should be done with sbt's standard `publish` task. */
 lazy val publishDists = taskKey[Unit]("Publish to ./dists/maven-sbt.")
 
+lazy val publishDestination = settingKey[Option[Resolver]]("determines publishTo in multiple places")
```

Comment:
@SethTisue  What is the reason for using this over publishTo? It is only referenced in one place (for setting publishTo)